dashboard/tests/test_dashboard.py

```diff
@@ -1262,5 +1263,50 @@ def f():
         requests.get("http://localhost:8265")
 
 
+@pytest.mark.skipif(
+    os.environ.get("RAY_MINIMAL") == "1" or os.environ.get("RAY_DEFAULT") == "1",
```

Comment:
why not RAY_DEFAULT?

Reply:
I thought the whole metrics things were not in default. removed.

Reply:
oh, I don't know actually, just asking :)